test/common/common.go

```diff
@@ -212,3 +216,31 @@ func OutputDebugInfo(testName string, kubeconfig string, additionalResources ...
 		_, _ = utils.KubectlWithOutput("get", resource, "--all-namespaces", "-o", "yaml", "--kubeconfig="+kubeconfig)
 	}
 }
+
+// IsMinimumVersion detects OCP versions given an x.y version lower bound
+func IsMinimumVersion(minVersion string) bool {
```

Comment:
Nit: `IsAtLeastVersion` or `IsVersionOrAbove` might be more descriptive - at first glance I might guess this function only checked for a specific version.